### PR TITLE
feat(our work): add the single project pages

### PR DESCRIPTION
**Pull request by Fdom92**
https://openforge.teamwork.com/#/tasks/17455100

### CODE REVIEW

**Comment by Wholloran** [comment]
Desktop: 
- Landing Page - The "our Work" is not centered in the graphic (desktop)
- Mobile Measures - The middle MM phone is a different style/size
- Mobile Measures - Decrease the white space between the "get back in motion" text and the asset to it's left (this decrease applies to all pages) (Desktop)
- Mobile Measures - Increase the spacing between the Utility/medical/reference icons (this can also be applied to these icons on the other pages) (desktop/Tablet)
- Mobile Measures - The Key features subtext looks like it's the wrong font color / eight (All projects)
- Mobile Measures - Reduce the white space between the two projects under "more Projects" (This is just for chrome, the spacing looks fine on Safari/firefox)
- LoudCloud - The third phone screen is a different style/size than the other (the one of the background graphic) (chrome)
- Voyage - THe first phone is a different size/style of the second/third phones (the ones on the background image)

Tablet:
- Tablet- The hero images on Tablet look stretched (the project-specific landing pages)
- Tablet - There is a lot of white space on the right side fo the entire page

Mobile:
- Mobile: the sliders for the three images are lines, per the marvel
 they should be circles 
- Mobile: Under "More projects" the title of the app should be above the image of it. Currently, it is under the app store links
- Mobile - Voyage - The hero image, text, and app icon aren't displaying on mobile (Safari)

- Landing Page - None of the "who we have worked with" icons are displaying (desktop Chrome/safari, Mobile Chrome). When they display on firefox the assets are not correct. (they do not fully show are only partially show 

Screen Shot 2020-01-07 at 1 15 05 PM

**Comment by luiskcs89** [comment]
Ready for another round

**Comment by Wholloran** [comment]
- ic@3401 & NG Atlanta logo still not showing correctly
- “The who we have worked with” icons are too large
- Mobile - The three phone slider is not centered
- Mobile - The voyage title ” get rewarded for your lifestyle” is on three lines and should only be on two

Screen Shot 2020-01-08 at 11 06 26 AM

**Comment by luiskcs89** [comment]
ready

**Comment by Wholloran** [comment]
QA Passes

File: src/pages/app-our-work-single/app-our-work-single.tsx [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+          <div class="key-features-text">
+            <div class="container">
+              <h2>{this.projects[this.match.params.project].features.title}</h2>
+              <p>{this.projects[this.match.params.project].features.text}</p>
+            </div>
+          </div>
+        </section>
+        <section class="more-projects container">
+          <h2 class="aside-lines">
+            <span>More Projects</span>
```
Comment (PaulPaulDevelops) [inline src/pages/app-our-work-single/app-our-work-single.tsx]: This needs to be added to our translation file.

File: src/pages/app-our-work-single/app-our-work-single.scss [hunk 2]
```diff
@@ -0,0 +1,424 @@
...
+
+  section.more-projects {
+    margin-top: 120px;
+
+    @include media-breakpoint-down(sm) {
+      margin-top: 40px;
+    }
+
+    h2 {
+      color: #292a2d;
```
Comment (PaulPaulDevelops) [inline src/pages/app-our-work-single/app-our-work-single.scss]: Let's use the global variable name for this, I believe we have it. If not, let's create it.

File: src/pages/app-our-work-single/app-our-work-single.scss [hunk 3]
```diff
@@ -0,0 +1,424 @@
...
+
+        @include media-breakpoint-down(sm) {
+          font-size: 28px;
+          text-align: left;
+        }
+      }
+
+      p {
+        margin-top: 35px;
+        color: #ffffff;
```
Comment (PaulPaulDevelops) [inline src/pages/app-our-work-single/app-our-work-single.scss]: Let's use the global variable name for this.

File: src/pages/app-our-work-single/app-our-work-single.scss [hunk 4]
```diff
@@ -0,0 +1,424 @@
...
+  section.key-features {
+    margin-top: 80px;
+
+    .key-features-text {
+      background-color: #292a2d;
+      padding-bottom: 36px;
+      padding-top: 60px;
+
+      h2 {
+        color: #ffffff;
```
Comment (PaulPaulDevelops) [inline src/pages/app-our-work-single/app-our-work-single.scss]: Let's use the global variable name for this.

File: src/pages/app-our-work-single/app-our-work-single.scss [hunk 5]
```diff
@@ -0,0 +1,424 @@
...
+        margin: 10px 20px 10px 0;
+      }
+    }
+  }
+
+  section.key-features {
+    margin-top: 80px;
+
+    .key-features-text {
+      background-color: #292a2d;
```
Comment (PaulPaulDevelops) [inline src/pages/app-our-work-single/app-our-work-single.scss]: Let's use the global variable name for this, I believe we have it. If not, let's create it.

File: src/pages/app-our-work-single/app-our-work-single.scss [hunk 6]
```diff
@@ -0,0 +1,424 @@
...
+
+      @include media-breakpoint-down(sm) {
+        text-align: left;
+        padding: 0 28px;
+      }
+    }
+
+    p {
+      margin-top: 30px;
+      color: #292a2d;
```
Comment (PaulPaulDevelops) [inline src/pages/app-our-work-single/app-our-work-single.scss]: Let's use the global variable name for this, I believe we have it. If not, let's create it.

File: src/pages/app-our-work-single/app-our-work-single.scss [hunk 7]
```diff
@@ -0,0 +1,424 @@
...
+          @include media-breakpoint-down(sm) {
+            top: 80px;
+            left: 80px;
+            right: initial;
+          }
+        }
+      }
+    }
+    h2 {
+      color: #333333;
```
Comment (PaulPaulDevelops) [inline src/pages/app-our-work-single/app-our-work-single.scss]: Let's use the global variable name for this, I believe we have it. If not, let's create it.

File: src/pages/app-our-work-single/app-our-work-single.scss [hunk 8]
```diff
@@ -0,0 +1,424 @@
...
+        }
+
+        @include media-breakpoint-down(sm) {
+          width: 99%;
+          font-size: 48px;
+        }
+      }
+
+      p {
+        color: #ffffff;
```
Comment (PaulPaulDevelops) [inline src/pages/app-our-work-single/app-our-work-single.scss]: Let's use the global variable name for this.

File: src/pages/app-our-work-single/app-our-work-single.scss [hunk 9]
```diff
@@ -0,0 +1,424 @@
...
+      flex-direction: column;
+      top: 80px;
+      align-items: center;
+
+      @include media-breakpoint-down(lg) {
+        top: 40px;
+      }
+
+      h1 {
+        color: #ffffff;
```
Comment (PaulPaulDevelops) [inline src/pages/app-our-work-single/app-our-work-single.scss]: Let's use the global variable name for this.

**Comment by luiskcs89** [comment]
ready

**Pull request merged**